src/odemis/driver/actuator.py

```diff
@@ -1757,9 +1758,8 @@ def __init__(self, name, role, children, axis_name, cycle=2*math.pi, inverted=No
         if not self._cycle >= 0:
             raise ValueError("Cycle needs to be a positive number. Got value %s." % self._cycle)
 
-        # just an offset to reference switch position
-        # 5% of cycle value should be sufficient to take care of the accumulated rotation error
-        self._offset = self._cycle*0.05
+        # just an offset to reference switch position using the shortest move
+        self._ref_start = ref_start
```

Comment:
Please add a check that `ref_start` is within the range of the child axis (or maybe even better, in this component axis)